File: go/sp.go
package main
import (
"fmt"
"strings"
)
func printPyramid(height int){
	for i := 1; i<=height; i++ {
		fmt.Printf("%s%s\n", strings.Repeat(" ", height-i), strings.Repeat("*", 2*i-1))
	}
}
func main(){
	var height int
	fmt.Print("Enter the height of pyramid: ")
	_, err:=fmt.Scanf("%d", &height)
	if err != nil{
		fmt.Println("Error:", err)
		return
	}
	printPyramid(height)
}

File: scala/sp.scala
object Pyramid {
  def printPyramid(height: Int): Unit = {
    for(i <- 1 to height){
      // print spaces
      for(j <- 1 to height){
        print(" ")
      }

      //Print stars
      for(k<-1 to 2*i-1){
        print("*")
      }

      // Move to the next line
      println()
    }
  }

  def main(args: Array[String]): Unit = {
    val height = 5 // Change the height of the pyramid as needed
    printPyramid(height)
  }
}